src/content/docs/more-integrations/open-source-telemetry-integrations/atlassian/atlassian-integration.mdx

```diff
@@ -0,0 +1,128 @@
+---
+title: Atlassian monitoring integration
+tags:
+  - Integrations
+  - Open source telemetry integrations
+  - OpenTelemetry
+  - Atlassian
+  - Quickstart
+metaDescription: Monitor Atlassian with New Relic.
+freshnessValidatedDate: 2024-04-12
+---
+
+import opentelemetryAtlassianNRapiToken from 'images/opentelemetry_screenshot-crop_connect-nr-jira.webp'
+
+import opentelemetryJiraIncidentManagement from 'images/opentelemetry_screenshot-full_jira-incident-mgt.webp'
+
+import opentelemetryJiraNRConfiguration from 'images/opentelemetry_screenshot-full_jira-nr-configuration.webp'
+
+
+
+New Relic offers an integration with Atlassian Jira that enables the incident tab feature of Jira. This feature syncs all your New Relic issue and incident information with Atlassian. This data will then appear in Jira on the incidents tab and will let you connect it with Jira issues to improve your outage workflows. The sections outline how you set up this integration and the best practices for usage.
+
+<Callout title="PREVIEW FEATURE">
+  This feature is still in preview, but we encourage you to try it out!
+</Callout>
+
+
+## Data sharing [#data-sharing]
+
+Once you install the integration, you allow New Relic for Jira to perform these actions:
+
+* Read entities, incidents, and issues from Jira.
+* Write entities, incidents, and issues to Jira.
+* Delete entities, incidents, and issues from both New Relic and Jira to mirror data reporting with each entity.
+
+New Relic data may be visible to non-New Relic users within your organization. These users can view information through the Jira application.
```

Comment:
Is this necessary with the callout below? It feels like they say the same thing essentially

Reply:
No, I don't think it's necessary. I'm going to remove it. Thanks!